Dockerfile

```diff
@@ -20,4 +20,4 @@ ENV PYTHONPATH=/app
 COPY agent /app/agent
 COPY ostorlab.yaml /app/agent/ostorlab.yaml
 WORKDIR /app
-CMD ["python3", "/app/agent/whatweb_agent.py"]
+CMD ["python3", "/app/agent/whatweb.py"]
```

Comment:
Why was it renamed to whatweb only, all the other agents, the main agent class has the name agent in it.